ChessCrafterDesktop/src/main/resources/GuiStrings_en_US.properties

```diff
@@ -130,6 +130,12 @@ Driver.startNewGame=Start a new Game
 Driver.sureYouWannaQuit=Are you sure you want to Quit?
 Driver.variants=Variants
 Driver.viewCompleted=View Completed Game
+GamePanel.inCheck=You are in Check!

```

Comment:
What if you're looking at someone else's team panel? Then it's not "You" are in check. How about we just say "Check" for this string?